engine/src/main/resources/org/hibernate/validator/ValidationMessages_nl.properties

```diff
@@ -1,8 +1,8 @@
 javax.validation.constraints.AssertFalse.message        = moet onwaar zijn
-javax.validation.constraints.AssertTrue.message         = moet waar zijn sein
+javax.validation.constraints.AssertTrue.message         = moet waar zijn
 javax.validation.constraints.DecimalMax.message         = moet kleiner dan ${strict == false ? 'of gelijk aan ' : ''}{value} zijn
 javax.validation.constraints.DecimalMin.message         = moet groter dan ${inclusive == true ? 'of gelijk aan ' : ''}{value} zijn
-javax.validation.constraints.Digits.message             = numerieke waarde ligt buiten het toegestane bereik (<{integer} cijfers>,<{fraction} cijfers> verwacht)
```

Comment:
'toegestane' was correct so please revert this line change

Reply:
done, thanks. I was unsure but it looked very odd :)